includes/managers/controls.php

```diff
@@ -862,6 +875,28 @@ public function remove_control_from_stack( $stack_id, $control_id ) {
 		return true;
 	}
 
+	/**
+	 * Should clear stack cache.
+	 * Whether the CSS requires to clear the controls stack cache.
+	 * @since 3.13.0
+	 * @access protected
+	 * @return bool True if the CSS requires to clear the controls stack cache, False otherwise.
+	 */
+	public function stack_cache_has_been_cleared() {
+		return $this->has_been_cleared;
+	}
```

Comment:
I would change the name of the private variable and the method to accurately describe what it does.
The Controls Manager shouldn't be aware of CSS, it should exist as a separate entity.

Also would need to be changed in the Pro PR.

```suggestion
	/**
	 * Has Stacks Cache Been Cleared.
	 *
	 * @since 3.13.0
	 * @access protected
	 * @return bool True if the CSS requires to clear the controls stack cache, False otherwise.
	 */
	public function has_stacks_cache_been_cleared() {
		return $this->has_stacks_cache_been_cleared;
	}
```

Reply:
Done